apps/explorer/src/components/top-packages/TopPackagesCard.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useQuery } from '@tanstack/react-query';
+import { useMemo, useState } from 'react';
+
+import { ErrorBoundary } from '../error-boundary/ErrorBoundary';
+import { TopPackagesTable } from './TopPackagesTable';
+
+import { useEnhancedRpcClient } from '~/hooks/useEnhancedRpc';
+import { FilterList } from '~/ui/FilterList';
+import { Tab, TabGroup, TabList, TabPanel, TabPanels } from '~/ui/Tabs';
+
+export type DateFilter = '3D' | '7D' | '30D';
+export type ApiDateFilter = 'rank3Days' | 'rank7Days' | 'rank30Days';
+export const FILTER_TO_API_FILTER: Record<DateFilter, ApiDateFilter> = {
+    '3D': 'rank3Days',
+    '7D': 'rank7Days',
+    '30D': 'rank30Days',
+};
+
+export function TopPackagesCard() {
+    const rpc = useEnhancedRpcClient();
+    const [selectedFilter, setSelectedFilter] = useState<DateFilter>('3D');
+
+    const { data, isLoading } = useQuery(
+        ['top-packages', selectedFilter],
+        async () => rpc.getMoveCallMetrics()
+    );
+
+    const filteredData = useMemo(() => {
```

Comment:
does this need to be memo-ized?

Reply:
not really, just carried it over from some previous code. I can remove.